README.md

```diff
@@ -30,6 +30,16 @@ $ git clone git@github.com:ministryofjustice/correspondence_tool_staff.git
 $ cd correspondence_tool_staff
 ```
 
+#### Generating Documentation
```

Comment:
can we add in the command to view documentation locally?

Reply:
Added.